ci/ci.sh

```diff
@@ -32,14 +32,26 @@ RUST_BACKTRACE=1 cargo test
 ###########
 # C Tests #
 ###########
-# Build the driver
-cargo build
+
+# Install and start the TPM server and Parsec
+cargo install parsec-service --features tpm-provider
+tpm_server &
+sleep 5
+tpm2_startup -c -T mssim 2>/dev/null
+tpm2_changeauth -c owner tpm_pass 2>/dev/null
+parsec-service --config ci/config.toml
+sleep 5
+
 # Compile Mbed Crypto (use the one in OUT_DIR)
-MBED_TLS_PATH=`find target -name "mbedtls-mbedtls-*"`
-pushd $MBED_TLS_PATH
+git clone https://github.com/ARMmbed/mbedtls.git
+pushd mbedtls
```

Comment:
I'd try to avoid trying to use the latest development branch as the base for our CI - it can start failing randomly for some issue that isn't even related to our code. Plus if they make some big change to the psa-crypto interface, we wouldn't be able to link to it.

Reply:
Maybe move to a set commit and work on that as a default until we get releases with Mbed Crypto in them?

Reply:
Yes, good point, I will select a specific tag.